source/jobs/sync-jobs.txt

```diff
@@ -54,6 +54,18 @@ To enable idempotent inserts for documents and embedded arrays, set
 ``user.properties`` file. Enabling idempotency may have substantial 
 performance impact on large jobs.
 
+Indexes
+~~~~~~~
+
+Relational Migrator creates new indexes in your MongoDB collection to
+supplement the migration process. For best performance, review the
+indexes in your collection after migration to ensure that they
+best fit your query patterns. 
+
+Relational Migrator does not create new indexes in the relational
+database. 
+
```

Comment:
(suggestion) 
It might be helpful to link out to the [Indexes](https://www.mongodb.com/docs/manual/indexes/) section in the manual. 

It might also be helpful to link specifically to the create, drop, and get indexes reference pages.

Reply:
Done!